tools/appveyor/build.bat

```diff
@@ -12,11 +12,17 @@ REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 REM See the License for the specific language governing permissions and
 REM limitations under the License.
 
-REM TODO: Get everything to build on Windows:
-REM bazel build //...
+REM We need this to append a variable within a loop.
+setlocal enabledelayedexpansion
+
+REM TODO: Is there an easier way to convert lines to a space-separated list?
+SET BUILDABLES=
+FOR /F usebackq %%T IN (`bazel query "kind(rule, //...)" ^| FINDSTR /C:"\:_" /V`) DO (
+    SET BUILDABLES=!BUILDABLES! %%T
+)
 
 REM TODO: Remove --output_user_root after https://github.com/bazelbuild/bazel/issues/4149 is fixed.
-bazel --output_user_root=c:/t/ build //opencensus/trace
+bazel --output_user_root=c:/t/ build %BUILDABLES%
```

Comment:
This PR is great overall, but I'd prefer to not have appveyor failing while we work on other stuff.

Please do:
```
bazel --output_user_root=c:/t/ build //opencensus/trace
￼REM bazel --output_user_root=c:/t/ build %BUILDABLES%
```

And we'll incrementally add targets as we get them building on Windows.

(I'm worried if every PR and commit has a red X after it, people will stop paying attention to CI entirely)

Reply:
Added. (and //opencensus/stats, which builds once the switch fallthrough errors were suppressed.)

Reply:
Oh, awesome! Thanks for that.